angular-workspace/.storybook/main.ts

```diff
@@ -38,15 +38,15 @@ const config: StorybookConfig = {
         transcludeMarkdown: true,
       },
     },
-    getAbsolutePath("@storybook/addon-links"),
-    getAbsolutePath("@storybook/addon-essentials"),
-    getAbsolutePath("@storybook/addon-interactions"),
-    getAbsolutePath("@storybook/addon-a11y"),
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "@storybook/addon-a11y",
   ],
   previewHead: (head) => `
     ${head}
-    <link rel="stylesheet" href="dso-toolkit/dist/dso.css">
-    <script src="iframe-resizer/child/index.umd.js"></script>
+    <link rel="preload" href="/Asap-Italic-VariableFont_wdth,wght.ttf" as="font" type="font/ttf" crossorigin>
+    <link rel="preload" href="/Asap-VariableFont_wdth,wght.ttf" as="font" type="font/ttf" crossorigin>
```

Comment:
In de Angular Storybook:
- `dso.css` is verplaatst naar `angular.json`
- `preload` path is meteen op root. Ik vermoed dat dit door de Angular bundler wordt geregeld.
- Geen `di.scss`: Dit is alleen voor HTML/CSS componenten nodig.

iframe-resizer child is naar preview.ts verplaatst. Dit geldt voor alle Storybooks.